ScreenManager.c

```diff
@@ -363,6 +363,11 @@ void ScreenManager_run(ScreenManager* this, Panel** lastFocus, int* lastKey, con
          ScreenManager_resize(this);
          continue;
       }
+      #ifdef KEY_FOCUS_IN
```

Comment:
Why have this in `#ifdef` here when it's without conditional checks at the other location where it's used?

Reply:
That's a leftover from making it more easy to test. We are quite inconsistent in what we have #ifdef'd in the keyboard / mouse handling and what not. But that's for another time. I'm happy to remove this #ifdef before merge.
Would be interesting if checking for these "keys" has any collateral damage in non-xterm use case. Unfortunately we typically only get wider testing when releases happen so ... YOLO.

Reply:
removed the #ifdef